src/lib/drupal/staticPaths.ts

```diff
@@ -1,80 +1,60 @@
 import { drupalClient } from '@/lib/drupal/drupalClient'
-import { getAllPagedListingPaths } from '@/lib/drupal/listingPages'
-import RESOURCE_TYPES from '@/lib/constants/resourceTypes'
-import { JsonApiResponse, JsonApiResourceWithPath } from 'next-drupal'
-
-async function getAllResourcesByResourceType(
-  resourceType: string
-): Promise<JsonApiResourceWithPath[]> {
-  const pageSize = 50 //must be <= 50 due to JSON:API limit
-  const params = {
-    // Note: we only need `path`, but we can't put it first:
-    // See:
-    //  https://next-drupal.org/guides/page-limit
-    //  https://dsva.slack.com/archives/C01SR56755H/p1695244241079879?thread_ts=1695070010.697129&cid=C01SR56755H
-    [`fields[${resourceType}]`]: 'title,path',
-    'page[limit]': pageSize,
+import {
+  ListingResourceTypeType,
+  getAllPagedListingStaticPathResources,
+  isListingResourceType,
+} from '@/lib/drupal/listingPages'
+import { RESOURCE_TYPES, ResourceTypeType } from '@/lib/constants/resourceTypes'
+import { queries } from '@/data/queries'
+import { StaticPathResourceType } from '@/types/index'
+import {
+  bifurcateLovellFederalPathResources,
+  removeLovellFederalPathResources,
+} from '@/lib/utils/lovell'
+
+/**
+ * Returns a static-path resource collection that is modified per business logic.
+ * Resources are added, removed and/or edited.
+ *
+ * E.g.
+ * - Resources are ADDED for subsequent listing pages (e.g. `stories/page-2`)
+ * - Resources are ADDED for Lovell bifurcation (Federal pages become VA *and* TRICARE)
+ * - Lovell Federal listing pages are REMOVED
+ */
+async function modifyStaticPathResourcesByResourceType(
+  resourceType: ResourceTypeType,
+  resources: StaticPathResourceType[]
+): Promise<StaticPathResourceType[]> {
+  if (resourceType === RESOURCE_TYPES.STORY) {
```

Comment:
assuming in the future most content types would get added here, except for the listing types which are handled below?

Reply:
Probably, though it's really a matter of what business cases we need to handle (both known cases that still need to be implemented, and unknown future cases). It might be the case that something needs to be run on _everything_, for example, in which case we wouldn't need to explicitly have a condition for every resource type. In that case, we'd have a call outside any condition. It's hard to say, but this implementation handles our current business logic.